yt_dlp/extractor/easybroadcast.py

```diff
@@ -0,0 +1,91 @@
+
+from yt_dlp.extractor.common import InfoExtractor
+from yt_dlp.utils import (
+    ExtractorError,
+)
```

Comment:
- use relative paths
- remove unnecessary line breaks and blank lines

please also apply these to  your other PR
```suggestion
from .common import InfoExtractor
from ..utils import ExtractorError
```